### DISCUSSION TITLE
Can't recognise my student email, even though the uni is recognised

### DISCUSSION THREAD

**Discussion by Kris-Martin** [community/community#54625] [GitHub Education]
### Hi everyone,

Hi,

Github Education has the wrong format for student emails from UNE (University of New England).

Github Education expects the following format:
student@une.edu.au

The correct format for students is:
student@myune.edu.au

Please update this so UNE students can successfully verify their student status. Please also note that many students from UNE study online (by distance) this is shown on their enrolment as online - the word distance is not used. This is also causing problems with verifying student status.

Thanks,

Kris Martin

**Comment by samus-aran** [comment]
Hi @Kris-Martin 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Although we prefer academic emails, you may proceed with alternative pieces of evidence to verify your student status.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

It would also be recommended that either you or a faculty member provides evidence that students do not receive academic emails to help with the request. 👍
